src/SixLabors.Fonts/Unicode/GraphemeClusterClass.cs

```diff
@@ -8,22 +8,124 @@ namespace SixLabors.Fonts.Unicode
     /// Unicode Grapheme Cluster classes.
     /// <see href="https://www.unicode.org/reports/tr29/#Grapheme_Cluster_Break_Property_Values"/>
     /// </summary>
-    internal enum GraphemeClusterClass
+    public enum GraphemeClusterClass
     {
+        /// <summary>
+        /// This is not a property value; it is used in the rules to represent any code point.
+        /// </summary>
         Any = 0,
+
+        /// <summary>
+        /// U+000D CARRIAGE RETURN (CR)
+        /// </summary>
         CR = 1,
```

Comment:
Minor point: I like what you did with the other enums by making the names much more clearer to their purpose e.g. CR here could become CarriageReturn.

Reply:
Yeah... If I make that public I'll definitely be doing that. Just trying to minimize work for now.

Reply:
The diff at line 11 shows it is now public?

Reply:
Ahhh.... I better double check that then!

Reply:
I remember now. I'd already tweaked `RegionalIndicator`. The problem was that I couldn't find longer definitions for some of the Hangul Symbols `LV`, `LVT` so thought best to do the minimal amount of changes. If we can find a definition I'm happy to do so.

Reply:
It's LeadingVowelTail!!

http://gernot-katzers-spice-pages.com/var/korean_hangul_unicode.html

Reply:
Good find! I fully get the approach to being more consistent with naming here. Your XML documentation makes it clear what is happening so if you can't find all the names I can't imagine the abbreviations really are an issue

Reply:
I can try to look for some names myself too but I'll be honest this isn't an area I know much about

Reply:
I've updated it now. 😃

Reply:
Looks great 👍